pywt/multidim.py

```diff
@@ -192,7 +192,7 @@ def idwt2(coeffs, wavelet, mode='sym'):
     for rowL, rowH in zip(L, H):
         data.append(idwt(rowL, rowH, wavelet, mode, 1))
 
-    return np.array(data, np.float64)
```

Comment:
This needs to go anyway, for `float32` support. We should probably test this - make sure any transform on `float64` always returns `float64` (this is important, precision is lost otherwise), and less importantly the same for `float32`